lib/train/utils.py

```diff
@@ -357,17 +357,17 @@ def get_hparams(init=True):
         help="if caching the dataset in GPU memory, 1 or 0",
     )
 
-    args = parser.parse_args()
+    args = parser.parse_args(args=args_list)
     name = args.experiment_dir.split('/')[-1]
     experiment_dir = os.path.join("./logs", args.experiment_dir)

```

Comment:
these logs are out of /tmp no?

Reply:
This appears to have an effect only on local directories:
```
>>> os.path.join("./logs", "/tmp/what/ever")
'/tmp/what/ever'
>>> os.path.join("./logs", "what/ever")
'./logs/what/ever'
```
In out case, we have
```
args.experiment_dir: `/tmp/tmpx0g40o72/rvc/logs/5528fa79-b008-4ad3-816d-38d50780e033`
experiment_dir: `/tmp/tmpx0g40o72/rvc/logs/5528fa79-b008-4ad3-816d-38d50780e033`
```

Reply:
ok I didn't reallize that the rvc code itself is being copied to tmp.
let's fix that as well but not in this PR

Reply:
Not sure what you want to do here. Hat's the problem specifically?